packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -24,11 +24,20 @@ export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<vo
   await initBLS();
 
   const {beaconNodeOptions, config} = await initializeOptionsAndConfig(args);
-  await persistOptionsAndConfig(args, beaconNodeOptions, config);
+  await persistOptionsAndConfig(args, config);
```

Comment:
Note that the bug in Pyrmont was caused by reading the config object, not the beaconNodeOptions